apps/store/extensions/app/social-reviews/themes/store/js/custom/review-sort.js

```diff
@@ -26,32 +26,53 @@ var usingTemplate = function(callback) {
         callback(Handlebars.partials['activity']);
     });
 };
-var redrawReviews = function(sortBy, callback) {
+var redrawReviews = function (sortBy, callback) {
     $('.com-sort .selected').removeClass('selected');
     var url = caramel.url('/apis/user-reviews');
     $.get(url, {
         target: target,
         sortBy: sortBy,
         offset: 0,
         limit: 10,
-        cacheBuster:generateCacheBuster()
-    }, function(obj) {
+        cacheBuster: generateCacheBuster()
+    }, function (obj) {
         var reviews = obj || [];
-        usingTemplate(function(template) {
+        usingTemplate(function (template) {
             var str = "";
             for (var i = 0; i < reviews.length; i++) {
                 var review = reviews[i];
-                str += template(review);
+                if (!review.isMyComment) {
+                    str += template(review);
+                }
+            }
+            if (str) {
+                $sort.removeClass('com-sort-hidden');
+                $('#'+sortBy).addClass('selected');
+                $stream.html(str);
+                $('.load-more').attr("value", 10);
+                $more.show();
+                $empty_list.text("");
+            }
+            else {
```

Comment:
Thank you, I have fixed this issue in following [commit](https://github.com/wso2/carbon-store/pull/643/commits/358a87f6fa0ffb067480daf1fe3e463bed87080e)